app/src/main/java/com/gst/gusto/review/viewmodel/ReviewViewModel.kt

```diff
@@ -0,0 +1,119 @@
+package com.gst.gusto.review.viewmodel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.gst.gusto.api.ApiResponse
+import com.gst.gusto.model.InstaReview
+import com.gst.gusto.repository.AuthRepositoryImpl
+import com.gst.gusto.repository.ReviewsRepositoryImpl
+import com.gst.gusto.util.GustoApplication
+import kotlinx.coroutines.launch
+
+class ReviewViewModel(
+    private val reviewsRepository: ReviewsRepositoryImpl,
+    private val authRepository: AuthRepositoryImpl
+) : ViewModel() {
+
+    private val xAuthToken = GustoApplication.prefs.getSharedPrefs().first
+    private val refreshToken = GustoApplication.prefs.getSharedPrefs().second
+
+    private val _instaReviews = MutableLiveData<MutableList<InstaReview?>>()
+    val instaReviews : LiveData<MutableList<InstaReview?>> = _instaReviews
```

Comment:
그러니까 얘도! MutableList 일 타입이 없겠죠! 
그리고 여기는 로딩 아이템을 넣어줄 것도 아니니까 non-null 로요!
```suggestion
    private val _instaReviews = MutableLiveData<List<InstaReview>>()
    val instaReviews : LiveData<List<InstaReview>> = _instaReviews
```